### PR TITLE
Add PaylineBundle

### PR DESCRIPTION
**Pull request by lolautruche**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: lolautruche/payline-bundle/2.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "Lolautruche\\PaylineBundle\\LolautruchePaylineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "PAYLINE_MERCHANT_ID": "your_payline_merchant_id",
+        "PAYLINE_ACCESS_KEY": "your_payline_access_key",
```
Comment (nicolas-grekas) [inline lolautruche/payline-bundle/2.0/manifest.json]: New practice here: I think env var for secrets should be commented, in order to allow managing secrets with the new `secrets:*` commands
`"#PAYLINE_ACCESS_KEY": "your_payline_access_key",`

File: lolautruche/payline-bundle/2.0/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<bg=blue;fg=white>                                                 </>
+<bg=blue;fg=white> You just installed LolautruchePaylineBundle!    </>
+<bg=blue;fg=white> What's next?                                    </>
+<bg=blue;fg=white>                                                 </>
+
+  * <fg=blue>Configure</> Payline environment variables and settings in <comment>config/packages/payline.yaml</comment>.
```
Comment (nicolas-grekas) [inline lolautruche/payline-bundle/2.0/post-install.txt]: but the yaml file references env vars - so we should tell ppl to have a look at `.env` files instead isn't it?

File: lolautruche/payline-bundle/2.0/manifest.json (hunk 1 above)
Comment (lolautruche) [reply lolautruche/payline-bundle/2.0/manifest.json]: Won't this make it unavailable then? If so, that would generate an exception as these information are mandatory

**Comment by lolautruche** [comment]
Review ping @Nyholm :-)

File: lolautruche/payline-bundle/2.0/manifest.json (hunk 1 above)
Comment (nicolas-grekas) [reply lolautruche/payline-bundle/2.0/manifest.json]: One will get an exception when accessing the service while the env var is not set - but a random string is not better, wdyt? At compile time, this should make no difference as no env vars should be read then.
Can you give it a try?

**Comment by lolautruche** [comment]
🎉

**Pull request merged**